pilot/cmd/pilot-agent/options/options.go

```diff
@@ -152,4 +152,7 @@ var (
 	exitOnZeroActiveConnectionsEnv = env.Register("EXIT_ON_ZERO_ACTIVE_CONNECTIONS",
 		false,
 		"When set to true, terminates proxy when number of active connections become zero during draining").Get()
+
+	disableSdsServerEnv = env.Register("DISABLE_SDS_SERVER", false,
```

Comment:
In the usage comment of this flag, can you describe under which scenarios, this flag should be set as true?

Reply:
Fixed in commit: [5ea155585df4f50eaff0ab5bb8b370f1fccd8b8f](https://github.com/istio/istio/pull/41440/commits/5ea155585df4f50eaff0ab5bb8b370f1fccd8b8f)